src/explore-education-statistics-common/src/components/Tag.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  className?: string;
+  strong?: boolean;
+}
+
+const Tag = ({ children, className, strong = false }: Props) => {
+  const classes = classNames('govuk-tag', {
+    [className || '']: className,
```

Comment:
This can just be changed to `classNames('govuk-tag', className)`

Reply:
cool ta